plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -532,7 +536,7 @@ public WriteMapping toWriteMapping(ConnectorSession session, Type type)
             return WriteMapping.objectMapping(dataType, longDecimalWriteFunction((DecimalType) type));
         }
         if (type.equals(TIMESTAMP_SECONDS)) {
-            return WriteMapping.longMapping("timestamp(0)", trinoTimestampToOracleDateWriteFunction());
+            return WriteMapping.longMapping("date", trinoTimestampToOracleDateWriteFunction());
```

Comment:
Add a comment why we chose `date` over `timestamp(0)`.

Reply:
Added a comment. Let me know if the comment is unclear.